admin/app/javascript/solidus_admin/application.js

```diff
@@ -0,0 +1 @@
+console.log("Hello from application.js")
```

Comment:
Is this intentional to make sure it works, or can we remove it? :thinking:

Reply:
Yes, this is intentional, once we start adding controllers in the form of `component.js` files I think we can cleanup, and maybe tweak a few more things.